pandas/tests/io/test_common.py

```diff
@@ -154,10 +154,14 @@ def test_read_non_existant(self, reader, module, error_class, fn_ext):
             fr"\[Errno 2\] File o directory non esistente: '.+does_not_exist\.{fn_ext}'"
         )
         msg8 = fr"Failed to open local file.+does_not_exist\.{fn_ext}.?, error: .*"
-
+        msg9 = (
+            # Encountered in feather 2020-02-07
+            fr"\[Errno 2\] Failed to open local file '.*does_not_exist\.{fn_ext}'. "
+            r"Detail: \[errno 2] 没有那个文件或目录"
+        )
```

Comment:
Or we can just stop testing the message here?

Reply:
i would be on board for that, cc @MomIsBestFriend looks like there is a new external_error_raised; is that for roughly this purpose?

Reply:
> i would be on board for that, cc @MomIsBestFriend looks like there is a new external_error_raised; is that for roughly this purpose?

@jbrockmendel Yes, exactly!